src/CBot/CBotProgram.h

```diff
@@ -334,6 +334,12 @@ class CBotProgram
     const std::list<CBotFunction*>& GetFunctions();
 
     /**
+     * \brief Check if class with that name exists (public or local)
```

Comment:
This explanation is ambiguous, it sounds like this function will check both private classes in this program and the global list of public classes, which is not the case.